lib/src/boilerplate_suggestors/advanced_props_and_state_class_migrator.dart

```diff
@@ -42,33 +42,36 @@ class AdvancedPropsAndStateClassMigrator extends GeneralizingAstVisitor
     final extendsFromCustomClass = !extendsFromUiPropsOrUiState(node);
     final hasMixins = node.withClause != null;
     final parentClassName = node.extendsClause.superclass.name.name;
+    final parentClassTypeArgs =
+        node.extendsClause.superclass.typeArguments ?? '';
 
     final className = stripPrivateGeneratedPrefix(node.name.name);
+    final classTypeArgs = node.typeParameters ?? '';
     final newDeclarationBuffer = StringBuffer()
       ..write('\n\n')
       // Write a fix me comment if this class extends a custom class
       ..write(!extendsFromCustomClass
           ? ''
           : '''
           // FIXME:
-          //   1. Ensure that all mixins used by $parentClassName are also mixed into this class.
+          //   1. Ensure that all mixins used by ${parentClassName}Mixin are also mixed into this class.
```

Comment:
Why was this changed? ${parentClassName}Mixin can't "use" any mixins

Perhaps the wording isn't super clear, and it should be changed to:
```suggestion
          //   1. Ensure that all mixins used by the old ${parentClassName} are also mixed into this class.
```